code/__DEFINES/materials/basic.dm

```diff
@@ -0,0 +1,340 @@
+// Basic and tier-0 materials. These are typically of the lowest value, and the simplest/purest elements.
+
+#define MAT_hydro "h2"
+/datum/mat/hydrogen
```

Comment:
The DEFINES folder is used solely to store defines, not to store new datum or objects.